### PR TITLE
rtmp-services: Add Amazon IVS service

### PR DESCRIPTION
**Pull request by palana**
This is a smaller version of https://github.com/obsproject/obs-studio/pull/10634

### Description
Add Amazon IVS service entry with global DNS names

### Motivation and Context
Currently users of Amazon IVS customers have to use the custom service entry and enter both a URL and their stream key; with multitrack video they'd also have to enter a config URL (which is currently not exposed in the custom service UI)

With these changes users will generally only have to enter their stream key (and select "Global (RTMPS, recommended)" or "Global (RTMP)" depending on whichever the Amazon IVS customer wants their users to use)

### How Has This Been Tested?
Tested using various IVS channels created for testing purposes, some of which were allowlisted for multitrack video

### Types of changes
- New feature (non-breaking change which adds functionality)

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

File: plugins/rtmp-services/data/services.json [hunk 1]
```diff
@@ -2903,6 +2903,26 @@
...
+            "servers": [
+                {
+                    "name": "Global (RTMPS, recommended)",
+                    "url": "rtmps://ingest.global-contribute.live-video.net:443/app"
+                },
+                {
+                    "name": "Global (RTMP)",
+                    "url": "rtmp://ingest.global-contribute.live-video.net/app"
+                }
+            ],
```
Comment (RytoEX) [inline plugins/rtmp-services/data/services.json:2919]: I'm not sure if this is correct. I do see that we have at least one service with mixed RTMP/RTMPS that doesn't specify a protocol. @tytan652 @derrod ?

**Review by RytoEX — COMMENTED** [review]
Trying to clarify something about mixed RTMP/RTMPS service entries.

File: plugins/rtmp-services/data/services.json (hunk 1 above)
Comment (tytan652) [reply plugins/rtmp-services/data/services.json:2919]: https://github.com/obsproject/obs-studio/wiki/Service-Submission-Guidelines

> *(Required if not RTMP/RTMPS)* The streaming protocol that the service use

Don't specify if it's RTMP and RTMPS, it's the only multi-protocol exception.

TLDR: `"protocol"` is to be removed.

**Review by Fenrirthviti — APPROVED** [review]
LGTM outside the (currently at writing) unresolved conversations.

Once those are fixed, approved on my end.

File: plugins/rtmp-services/data/services.json (hunk 1 above)
Comment (RytoEX) [reply plugins/rtmp-services/data/services.json]: Fixed and force pushed.

**Review by RytoEX — APPROVED** [review]
